AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/GeneratePackageManifest.swift

```diff
@@ -32,8 +32,8 @@ struct GeneratePackageManifestCommand: ParsableCommand {
     @Option(help: "The names of the services to include in the package manifest. This defaults to all services located in aws-sdk-swift/Sources/Services")
     var services: [String] = []
 
-    @Flag(help: "If the package manifest should include the integration tests.")
-    var includeIntegrationTests: Bool = false
+    @Flag(help: "If the package manifest should include services with integration tests.")
+    var includeServicesWithIntegTests: Bool = false
```

Comment:
Why do we need this option at all in the main Package.swift if we have a whole separate Package.swift for integration tests now?

If we code-generate with the special sdk.properties file for integration tests, shouldn't that ensure that all the needed services for integration tests are present?

Reply:
That's a good point, for some reason I thought service targets will need to be available from the OG Package.swift for it to be available for integration tests, but as you said the source files are already there and I can just add targets to them in the new Package.swift anyways. Will update accordingly.

Reply:
Actually, it seems I can't add targets to generated SDK services from within `IntegrationTests/Package.swift`, because apparently you can't refer to files outside the package root (where Package.swift is located). It seems enabling services for integration tests must be done as it is by the OG Package manifest.

Reply:
Can we use symlinks to link the code?  Git will commit symlinks IIRC.

Reply:
@sichanyoo Confirmed, you don't need any of the integration test logic in the main Package.swift anymore.
See changes in https://github.com/awslabs/aws-sdk-swift/pull/1591 .

Reply:
#1591 doesw as I described at top: generate only the services used for integraion testing by using the special `sdk.properties` file to pick enabled services.  The logic in main Package.swift becomes superfluous as long as that properties file is maintained as tests and/or dependencies are changed.